### ISSUE TITLE
use existing *.fa.fai

### ISSUE THREAD

**Issue by housw** [josephhughes/DiversiTools#17]
Hi, 

I see you have a good reason to remove the existing index of the reference in issue #6, it's great to make the tool robust. While I'm actually facing another issue with deleting it since I'm profiling many bam files in parallel, and I'm not sure will it affect the other jobs when it's deleted in one job? Or could you please add one more option to reuse the existing `*.fai` file?

Thanks in advance!
